runtime/endpoint_config.ts

```diff
@@ -128,8 +129,8 @@ class EndpointConfig implements EndpointConfigData {
 			this.connect = DatexObject.get(<any>config, 'connect')
 			this.temporary = DatexObject.get(<any>config, 'temporary')
 			this.ws_relay = DatexObject.get(<any>config, 'ws_relay')
-			// TODO: enable nodes from cached .dx file, currently disabled for backends to avoid problems during development
```

Comment:
Enabled this again, this was only really a problem at the time when the private keys of the main unyt relay endpoints changed frequently.